src/main/java/com/happeningnow/controller/OrganizerController.java

```diff
@@ -0,0 +1,62 @@
+package com.happeningnow.controller;
+
+import com.happeningnow.model.Organizer;
+import com.happeningnow.repository.OrganizerRepository;
+import com.happeningnow.service.ServiceOrganizer;
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.ExampleMatcher;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/happening-now")
+public class OrganizerController {
+
+    private final ServiceOrganizer serviceOrganizer;
+
+    public OrganizerController(ServiceOrganizer serviceOrganizer, OrganizerRepository organizerRepository){
+        this.serviceOrganizer = serviceOrganizer;
+    }
+
+    @PostMapping("/save-organizer")
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
It is a cleaver way of stating the method's return Ressponse. But I would like you to use "ResponseEntity<>" since it will give us more flexibility in terms of building an Response

Reply:
Ok